core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -59,7 +59,7 @@ import scala.util.control.NoStackTrace
   * @param conf H2O configuration
   */
 class H2OContext private (@(transient @param @field) val sparkContext: SparkContext, @(transient @param @field) conf: H2OConf) extends Logging
-  with Serializable with SparkDataFrameConverter with SupportedRDDConverter with DatasetConverter
```

Comment:
I am not sure about it - this will hide `Serializable` but H2OContext still remains `Serializable`.